Scripts/gen_Digitize.sh

```diff
@@ -62,7 +67,9 @@ else
 fi
 echo "Generating digitization scripts for $primary conf $conf output $digout database purpose, version $dbpurpose, $dbver"
 
-rm digitize.fcl
+if [[ -f "digitize.fcl" ]]; then
```

Comment:
I think you would get the same result with one line
`rm -f digitize.fcl`

Reply:
good point.  I updated this script and all the rest for the same issue.